src/Apps/Order/Routes/__tests__/Status.test.tsx

```diff
@@ -1,32 +1,56 @@
 import { Message } from "@artsy/palette"
-import { mount, render } from "enzyme"
+import { render } from "enzyme"
 import React from "react"
-import { HeadProvider } from "react-head"
-import { Status_order } from "../../../../__generated__/Status_order.graphql"
+import { graphql } from "react-relay"
 import {
+  mockResolver,
   OrderWithShippingDetails,
   PickupOrder,
 } from "../../../../Apps/__test__/Fixtures/Order"
-import { StatusRoute } from "../Status"
+import { MockBoot, renderRelayTree, UntilCallback } from "../../../../DevTools"
+import { StatusFragmentContainer } from "../Status"
+
+jest.unmock("react-relay")
 
 describe("Status", () => {
-  const getWrapper = (order: Status_order, headTags = []) =>
-    mount(
-      <HeadProvider headTags={headTags}>
-        <StatusRoute order={order} />
-      </HeadProvider>
-    )
+  const getWrapper = (
+    order: any,
+    headTags: Array<React.ReactElement<any>> = [],
+    until: UntilCallback = null
```

Comment:
What does `UntilCallback` do?

Reply:
See the doc in `renderUntil` https://github.com/artsy/reaction/pull/1345/files#diff-99ca552957c29b34c115ada9663e43b7R33